ts/src/bybit.ts

```diff
@@ -4192,10 +4203,15 @@ export default class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
+        let market = this.market (symbol);
+        if (market['option']) {
+            this.options['loadAllOptions'] = true;
```

Comment:
same here

Reply:
@Dan-krm this is the same thing 😅 we cannot do things like that
we should:
- use amount/price to precision is precision is available
- don't format price/amount precision if those values are not available

Reply:
Okay it should be good now